src/styles/blocks/html/body.scss

```diff
@@ -0,0 +1,10 @@
+.html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the class name `.html` should not be used because HTML is a tag, not a class. Use a class name that reflects the purpose of the style or remove it if it's not necessary.